data/markdownText.yaml

```diff
@@ -0,0 +1,18 @@
+---
+example-markdown: |+
```

Comment:
this sure doesn't look like a YAML file. could we use a different mimetype? https://stackoverflow.com/questions/10701983/what-is-the-mime-type-for-markdown

Reply:
this is a yaml file that contains snippets of markdown, this is so that chunks of text can be named.

Reply:
oh! now i see it, yes. it's like a magic eye =). is the `---` at the top required? what does it do?

Reply:
the `---` indicates the start of a document, it is optional for the first document in a file

Reply:
huh, ok, thanks